assets/tsconfig.json

```diff
@@ -3,7 +3,7 @@
   "exclude": ["node_modules"],
   "compilerOptions": {
     /* Basic Options */
-    "target": "es5" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */,
+    "target": "ES2018" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */,
```

Comment:
Curious what difference this makes to us, since we only use `tsc` for type-checking and not to actually compile to JS.

Reply:
It's to eliminate an error [here](https://github.com/mbta/screens/blob/3e2d7f91379c8e680c89e410be8547fbcfd08c5f/assets/src/components/v2/subway_status/subway_status_common.tsx#L103). Named capture groups need a target of at least ES2018. Because we don't care about the target for compilation, it seems like a harmless change for resolving the "error".